node/typings/orderFormEvent.ts

```diff
@@ -0,0 +1,5 @@
+export interface OrderFormEvent {
+  eventName: string
+  eventObject: unknown
```

Comment:
This would allow you to use the spread syntax without casting it to `Object`:
```suggestion
  eventObject: Record<string, unknown>
```